cli/cmd/component-apply.go

```diff
@@ -41,6 +41,15 @@ When run with no arguments, all components listed in the configuration are appli
 	},
 }
 
+var debug bool
+
+// nolint:gochecknoinits
+func init() {
+	componentCmd.AddCommand(componentApplyCmd)
+	pf := componentApplyCmd.PersistentFlags()
```

Comment:
should we use 2 init functions? I am little confused, can you please give a little context?

Reply:
Oh, we shouldn't, nice catch! (I realized now that you can have more than one `init()` function per package :exploding_head: